app/lib/constants/colors.ts

```diff
@@ -156,7 +156,7 @@ const dark = {
 	buttonBackgroundPrimaryPress: '#245399',
 	buttonBackgroundPrimaryDisabled: '#1D3963',
 
-	buttonBackgroundSecondaryDefault: '#2F343D',
+	buttonBackgroundSecondaryDefault: '#404754',
```

Comment:
Can you show on DM where did you get this new value from?

Reply:
I just missclicked the copy button on figma but it should be updated  (I've just adjusted it), I will send to you on DM where I get the value.